test/e2e/windows/kubelet_stats.go

```diff
@@ -77,12 +77,20 @@ var _ = SIGDescribe("[Feature:Windows] Kubelet-Stats [Serial]", func() {
 						}
 						statsChecked = statsChecked + 1
 
-						framework.ExpectEqual(*podStats.CPU.UsageCoreNanoSeconds > 0, true, "Pod stats should not report 0 cpu usage")
-						framework.ExpectEqual(*podStats.Memory.WorkingSetBytes > 0, true, "Pod stats should not report 0 bytes for memory working set ")
-
+						if *podStats.CPU.UsageCoreNanoSeconds <= 0 {
+							framework.Failf("Pod stats report cpu usage equal to %v but it should be greater than 0", *podStats.CPU.UsageCoreNanoSeconds)
+						}
+						if *podStats.CPU.UsageCoreNanoSeconds <= 0 {
+							framework.Failf("Pod stats report bytes for memory working set equal to %v but it should be greater than 0", *podStats.Memory.WorkingSetBytes)
+						}
 						for _, containerStats := range podStats.Containers {
-							framework.ExpectEqual(containerStats.Logs != nil, true, "Pod stats should have container log stats")
+							if containerStats.Logs == nil {
+								framework.Failf("Pod stats should have container log stats, but it is equal to %v", nil)
```

Comment:
Include the pod namespace+name and container name for which this fails.

Reply:
Can you confirm that in this case Pod name and namespace will be podStats.PodRef.Name and podStats.PodRef.Namespace

Reply:
If those are valid fields, then yes. I don't know this struct.